system/Test/bootstrap.php

```diff
@@ -84,20 +84,6 @@ class_alias('Config\Services', 'CodeIgniter\Services');
 // Initialize and register the loader with the SPL autoloader stack.
 Services::autoloader()->initialize(new Autoload(), new Modules())->register();
 
-// Now load Composer's if it's available
-if (is_file(COMPOSER_PATH)) {
-    /*
-     * The path to the vendor directory.
-     *
-     * We do not want to enforce this, so set the constant if Composer was used.
-     */
-    if (! defined('VENDORPATH')) {
-        define('VENDORPATH', realpath(ROOTPATH . 'vendor') . DIRECTORY_SEPARATOR);
-    }
-
-    require_once COMPOSER_PATH;
```

Comment:
Removing this changes the behavior?

Reply:
@paulbalandan I overlooked the deletion of `require_once COMPOSER_PATH;`.
I restored.